config/i18n/locales/english/translations.json

```diff
@@ -8,8 +8,8 @@
     "label": "Search",
     "accessible-name": "Submit your search query",
     "placeholder": {
-      "default": "Search our tutorials",
-      "numbered": "Search {{ roundedTotalRecords }}+ tutorials"
+      "default": "Search our news articles, tutorials and books",
+      "numbered": "Search {{ roundedTotalRecords }}+ news articles, tutorials and books"
```

Comment:
@moT01, what do you think of the copy? Which of the followings do you prefer?
Search our news articles, tutorials and books
Search our articles, tutorials, and books on /news
Search our articles, tutorials, and books on News
Search our articles, tutorials, and books on our publication
Search our articles, tutorials, and books on our News publication